README.md

```diff
@@ -186,7 +186,8 @@ do your own research!
 
 * [Citadel](https://citadel.li/)
 * [GDEX](https://www.gdex.io/) 
-* [Rudex](https://rudex.org/)
+* [ioBanker](https://iobanker.com/) - Stablecoins too! (HONEST.BTC, .CNY, .ETH, .USD, .XAG, .XAU, .XRP) 
```

Comment:
Why specially mention stablecoins for ioBanker? Other exchanges don't support stablecoins?

Reply:
Because ioBanker is now the only UI which features HONEST.Assets out of the box.

Reply:
Featured tokens of other UIs aren't listed in the page, IMO we either list for all or for none.